deepface/detectors/Yolov8faceWrapper.py

```diff
@@ -0,0 +1,71 @@
+from deepface.detectors import FaceDetector
+
+# Models names and paths
+PATHS = {
+    "yolov8n": "/.deepface/weights/yolov8n-face.pt",
+}
+
+# Google Drive base URL
+BASE_URL = "https://drive.google.com/uc?id="
+
+# Models' Google Drive IDs
+IDS = {
+    "yolov8n": "1qcr9DbgsX3ryrz2uU8w4Xm3cOrRywXqb",
```

Comment:
please do not use IDS here. you can directly use this is in weights.

Reply:
ok